content/pt-br/event-streaming.md

```diff
@@ -0,0 +1,30 @@
+---
+título: Streaming de Eventos
+estado: Concluído
+categoria: conceito
+---
+
+## O que é isso
```

Comment:
we have an agreement to use "O que é" instead of "O que é isso" to standardize titles